src/manager.cc

```diff
@@ -919,8 +919,8 @@ bool fastcat::Manager::SetActuatorPositions()
     }
     auto find_pos_data = actuator_pos_map_.find(dev_name);
 
-    MSG("Setting actuator: %s to saved pos: %lf", dev_name.c_str(),
-        find_pos_data->second.position);
+    //MSG("Setting actuator: %s to saved pos: %lf", dev_name.c_str(),
```

Comment:
Why don't leaving the message in with the understanding that the code does nothing for absolute encoders.